update (#1783)
cocos-creator/creator-docs: en/concepts/scene/shadow.md

Percentage Closer Filtering (PCF) is a simple, common technique used to achieve shadow edge desampling, by smoothing shadow edges to eliminate jaggedness in shadow mapping. The principle is to sample around the current pixel (also called a fragment), then calculate the ratio of the sample closer to the lights compared to the fragment, use this ratio to scale the scattered light and specular light, and then color the fragment to blur the shadow edges.

Cocos Creator currently supports hard, 4x(**SOFT** Mode), 9x(**SOFT_2X** Mode) sampler. The larger the magnification, the larger the sampling area and the more blurred the shadow edges.

#### AutoAdapt Adaptive Shadow Calculation

> AutoAdapt adaptive shadow calculation automatically calculates the extent of shadow generation in the lightView and the proximity of the shadow camera. This can be used to check if the hardware supports semi-floating point types, and if so, Creator will automatically turn on linear depth and use texture compression.

Reviewer: ```suggestion
AutoAdapt adaptive shadow calculation automatically calculates the extent of shadow generation in the lightView and the proximity of the shadow camera. This can be used to check if the hardware supports semi-floating point types, and if so, Creator will automatically enable linear depth or use texture compression.
```

> AutoAdapt adaptive shadow calculation automatically calculates the extent of shadow generation in the lightView and the proximity of the shadow camera. Starting from 3.3.0, the **linearDepth** and **Packing** options in the shadow options will be removed and the Creator will automatically determine the hardware capabilities and choose the best way to render the shadows.

Reviewer: ```suggestion
AutoAdapt adaptive shadow calculation automatically calculates the extent of shadow generation in the lightView and the proximity of the shadow camera.

Starting with v3.3, the **Linear** and **Packing** options for Shadows in the **Inspector** panel have been removed, and Creator will automatically determine the hardware capabilities and choose the best way to render the shadows.
```
